src/contour/TerminalWindow.cpp

```diff
@@ -205,11 +205,12 @@ void ScrollableDisplay::onValueChanged()
 }
 #endif // }}}
 
-TerminalWindow::TerminalWindow(config::Config _config, bool _liveConfig, string _profileName, string _programPath) :
+TerminalWindow::TerminalWindow(config::Config _config, bool _liveConfig, string _profileName, string _programPath, Controller &_controller) :
     config_{ std::move(_config) },
     liveConfig_{ _liveConfig },
     profileName_{ std::move(_profileName) },
-    programPath_{ std::move(_programPath) }
+    programPath_{ std::move(_programPath) },
+    controller_{ &_controller }
```

Comment:
```suggestion
    controller_{ _controller }
```
because of the pointer-to-ref suggestion above.